source/develop/release-management/features/ovirt-ovn-provider.md

```diff
@@ -0,0 +1,356 @@
+modifies the local OpenFlows on the host and deletes the chassis id from the
+bindings. Other OVN-controllers notice the binding change, and update their
+local flows.
+
+## Packaging and installation
+
+### OVN Central Server
+The OVN central server must be intalled manually. It must be accessible via network to
+the OVN provider and to all the hosts which use OVN.
+
+### OVN Controller
+The OVN Controller must be installed on each host using OVN.
+After installation the OVN controller must be configured with the value of the OVN central
+which it will be using and the local IP used for tunneling.
+This could be automated using an otopi plugin, but this would require a one-time configuration
+of the plugin. A way to establish the tunneling IP on the host should be provided, as well as
+the IP of the OVN central.
+A look at the plugin directory might be helpful to understand the process:
+`/usr/share/ovirt-host-deploy/plugins/ovirt-host-deploy`
+
+Another possibility would be to to use vdsm-tool configure.
+
+The OVN controller will probably be installed together with the VIF driver.
+
+### Provider packaging and installation
+The provider will be delivered as an RPM.
+The provider is the proxy between the oVirt engine, and the OVN north DB. The oVirt engine
+will need the IP of the provider to connect to it. The provider in turn will need the IP of
+the OVN north DB to be able to connect to it.
+The provider IP is specified whent adding a provider in the oVirt engine.
+The OVN north DB IP must be specified when starting the OVN provider. If this is not specified,
+the provider will assume the OVN north DB is running on the same host.
+
+Note that running oVirt engine, the provider and OVN north DB on different hosts is possible,
+the most common scenario will be running all three on the same host.
+
+### Driver packaging
+The VIF driver will be delivered as an rpm. As such, it will be installed manually
+on the hosts. The VIF driver connects to the local OVS instance and OVN north DB instance used
+by the local OVS. It needs no further configuration.
+
+The VIF driver RPM could be made available in the VDSM repo, or a local repo available to the hosts.
+It could then be installed by being added to:
+/usr/share/ovirt-host-deploy/plugins/ovirt-host-deploy/vdsmhooks/packages.d\
+All RPM files specified there will be installed on the host during the host deploy operation.
+There should be a separate file for each linux distribution. The file extension
+must be the name of the linux distribution on which the rpm's are to be installed
+(fedora, redhat, ...).
+
+### Installation issues.
+
+The provider and driver will need to access various system resources with limited access, such
+as sockets, network and file access.
+The following items must be taken care of to allow this:
+*   create required firewalld and iptables rules
+*   giving the `vdms` user (user executing hooks) access to `root` owned OVS resources
```

Comment:
vdms -> vdsm.

most hooks use sudoers. we should refrain of modifying openvswitch socket permission, since openvswitch upgrade/restart may ruin it.